mw-webapp/src/component/tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,42 @@
+import {ReactNode, useState} from "react";
+import styles from "src/component/tooltip/Tooltip.module.scss";
+
+const enum PositionTooltip {
+  TOP = "top",
+  LEFT = "left",
+  RIGHT = "right",
+  BOTTOM = "bottom"
+} // In my opinion, this should be a global one that controls the naming of props
+
+const enum ClassesTooltip {
+  MAIN = "main"
+} // In my opinion, this should be a global one that controls the naming of props
+
+interface TooltipProps {
+  children: ReactNode;
+  content: string;
+  styleComponent: ClassesTooltip;
```

Comment:
nice prop to handle different types of tooltip! But currently we need just something simple, get rid of it)

Reply:
Ok, fixed